### PR TITLE
Hotfix/flutter build

### PR DESCRIPTION
**Pull request by ultymatom**

### CODE REVIEW

File: scripts/pipelines/gcloud/pipeline_generator.sh [hunk 1]
```diff
@@ -149,7 +149,11 @@ function createTrigger {
       gcloud storage buckets create "gs://${gCloudProject}_cloudbuild" --project="${gCloudProject}"
     fi
     # We create the trigger
-    gcloud beta builds triggers create cloud-source-repositories --repo="$gCloudRepo" --branch-pattern="$branchTrigger"  --build-config="${pipelinePath}/${yamlFile}" --project="$gCloudProject" --name="$pipelineName" --description="$triggerDescription" --substitutions "${subsitutionVariable}${artifactPathSubStr}"
+    if [ "$previousPipelineyaml" == "" ]; then #Build trigger, executes when occurs a push to the repo
+        gcloud beta builds triggers create cloud-source-repositories --repo="$gCloudRepo" --branch-pattern="$branchTrigger"  --build-config="${pipelinePath}/${yamlFile}" --project="$gCloudProject" --name="$pipelineName" --description="$triggerDescription" --substitutions "${subsitutionVariable}${artifactPathSubStr}"
```
[the remark this answers is not in the document]
Comment (ultymatom) [reply scripts/pipelines/gcloud/pipeline_generator.sh:153]: @sonatype-lift ignore

**Review by lpinon — APPROVED** [review]

File: scripts/pipelines/gcloud/pipeline_generator.sh [hunk 2]
```diff
@@ -149,7 +149,11 @@ function createTrigger {
...
+    else #Not build trigger, executed manually
+        gcloud beta builds triggers create cloud-source-repositories --repo="$gCloudRepo" --branch-pattern="manualtrigger"  --build-config="${pipelinePath}/${yamlFile}" --project="$gCloudProject" --name="$pipelineName" --description="$triggerDescription" --substitutions "${subsitutionVariable}${artifactPathSubStr}"
```
Comment (albertdb) [inline scripts/pipelines/gcloud/pipeline_generator.sh:155]:
```suggestion
        gcloud beta builds triggers create cloud-source-repositories --repo="$gCloudRepo" --branch-pattern="_manual_"  --build-config="${pipelinePath}/${yamlFile}" --project="$gCloudProject" --name="$pipelineName" --description="$triggerDescription" --substitutions "${subsitutionVariable}${artifactPathSubStr}"
```

File: scripts/pipelines/gcloud/pipeline_generator.sh [hunk 3]
```diff
@@ -149,7 +149,11 @@ function createTrigger {
       gcloud storage buckets create "gs://${gCloudProject}_cloudbuild" --project="${gCloudProject}"
     fi
     # We create the trigger
-    gcloud beta builds triggers create cloud-source-repositories --repo="$gCloudRepo" --branch-pattern="$branchTrigger"  --build-config="${pipelinePath}/${yamlFile}" --project="$gCloudProject" --name="$pipelineName" --description="$triggerDescription" --substitutions "${subsitutionVariable}${artifactPathSubStr}"
+    if [ "$previousPipelineyaml" == "" ]; then #Build trigger, executes when occurs a push to the repo
```
Comment (albertdb) [inline scripts/pipelines/gcloud/pipeline_generator.sh:152]:
```suggestion
    if [ "$previousPipelineyaml" == "" ]; then # Initial trigger, executes when occurs a push to the repo
```

File: scripts/pipelines/gcloud/pipeline_generator.sh [hunk 4]
```diff
@@ -149,7 +149,11 @@ function createTrigger {
...
+        gcloud beta builds triggers create cloud-source-repositories --repo="$gCloudRepo" --branch-pattern="$branchTrigger"  --build-config="${pipelinePath}/${yamlFile}" --project="$gCloudProject" --name="$pipelineName" --description="$triggerDescription" --substitutions "${subsitutionVariable}${artifactPathSubStr}"
+    else #Not build trigger, executed manually
```
Comment (albertdb) [inline scripts/pipelines/gcloud/pipeline_generator.sh:154]:
```suggestion
    else # Chained trigger, executed manually
```

**Review by albertdb — CHANGES_REQUESTED** [review]

**Review by albertdb — APPROVED** [review]

**Review by lpinon — APPROVED** [review]

**Pull request merged**